docs/DevGuide/Notation.md

```diff
@@ -0,0 +1,134 @@
+  \f$P_N\f$ scheme and converges at order \f$\mathcal{O}\left(\Delta
+  x^{N+1}\right)\f$ where \f$\Delta x\f$ is the 1d size of the element.
+- The unit normal vector to one of the surfaces of the element is denoted by
+  \f$n_i\f$.
+- The numerical flux boundary correction term which includes the normal vectors
+  is denoted by \f$G(F^{i,+},n_i^+,u^+,F^{i,-},n_i^-,u^-)\f$ where the \f$+\f$
+  and \f$-\f$ denote the values on the outer and inner side of the mortar,
+  respectively. To give a concrete example, for the Rusanov flux we have
+  \f{align*}{
+      G^{\mathrm{Rusanov}}=
+      \frac{1}{2}\left(F^{i,+}n_i^+-F^{i,-}n_i^-\right)
+      -\frac{C}{2}\left(u^+-u^-\right)
+  \f}
+
+## General Relativity and (Magneto)hydrodynamics{#gr_hydro_notation}
+
+We use the following conventions and notation:
+- In general we work in units where \f$G=c=1\f$.
+- Latin letters at the beginning of the alphabet, \f$a,b,c,\ldots\f$ are
+  spacetime indices ranging from 0 to 3.
+- Latin letters \f$i,j,k,\ldots\f$ are spatial indices ranging from 1 to 3.
+
+The notation for general relativity quantities used is:
+- The Einstein summation convention is used where repeated indices are summed
+  over.
+- The spacetime metric is denoted by \f$g_{ab}\f$ with signature
+  \f$(-,+,+,+)\f$, and its determinant by \f$g\f$.
+- The lapse and shift are denoted by \f$\alpha\f$ and \f$\beta^i\f$,
+  respectively.
+- The spatial metric and its determinant are denoted by \f$\gamma_{ij}\f$ and
+  \f$\gamma\f$, respectively.
+- We write the line element as:
+  \f{align*}{
+      ds^2=g_{ab}dx^a dx^b=-\alpha^2 dt^2 +
+      \gamma_{ij}
+      \left(dx^i + \beta^i dt\right)\left(dx^j + \beta^j dt\right).
+  \f}
+- The unit timelike normal to the hypersurface is
+  \f$n^a=(1/\alpha,-\beta^i/\alpha)\f$ and \f$n_a=(-\alpha,0,0,0)\f$.
+- The extrinsic curvature is denoted by \f$K_{ij}\f$ and its trace by
+  \f$K\f$. We use the sign convention
+  \f{align*}{
+      K_{ab}=-\frac{1}{2}\mathcal{L}_n g_{ab}
+  \f}
+- For the generalized harmonic evolution system \f$\Pi_{ab}=-n^c\partial_c
+  g_{ab}\f$ and \f$\Phi_{iab}=\partial_i g_{ab}\f$.
+
+The notation for (magneto)hydrodynamics used is:
+- The fluid pressure, rest mass density, specific internal energy, and specific
+  enthalpy are denoted by \f$p\f$, \f$\rho\f$, \f$\epsilon\f$, and \f$h\f$,
+  respectively.
+- The 4-velocity of the fluid is \f$u^a\f$ and \f$u^a u_a=-1\f$.
+- The spatial velocity of the fluid as measured by an observer at rest in the
+  spatial hypersurfaces (“Eulerian observer”) is
+  \f{align*}{
+      v^i=\frac{1}{\alpha}\left(\frac{u^i}{u^0}+\beta^i\right)
+  \f}
+  with corresponding Lorentz factor
+  \f{align*}{
+      W=-u^a n_a=\alpha u^0=\frac{1}{\sqrt{1-\gamma_{ij}v^i v^j}}
+  \f}
+- The Faraday tensor is denoted by \f$F^{ab}\f$ and its dual is given by
+  \f$^{*}\!F^{ab}=\frac{1}{2}\epsilon^{abcd}F_{cd}\f$ where
+  \f$\epsilon^{abcd}\f$ is the Levi-Civita tensor.
```

Comment:
...and \epsilon^{0123} = +1/sqrt{-g}, or some other way to introduce the explicit sign convention. (Some GR textbooks define \epsilon_{0123} = +sqrt{-g}, which flips the sign of the fully contravariant \epsilon.)

Might also be better to define this at the beginning, after the metric conventions. Clearly the Levi-Civita tensor isn't restricted to hydro applications.

Reply:
Pushed a fixup. Please let me know if you are happy with the wording :)

Reply:
I'm happy with the change.

Reply:
I think we should be consistent in following the MTW conventions here, namely \epsilon_{0123} = +sqrt{-g}. The place it makes a difference for us is in the MHD definitions, and there does not seem to be consensus in the literature. Since all our other definitions are those of MTW, let's stick with those.